Dockerfile

```diff
@@ -249,26 +249,28 @@ RUN set -x \
 	&& go build -v -o /usr/local/bin/rsrc github.com/akavel/rsrc \
 	&& rm -rf "$GOPATH"
 
+# Upload docker source
+COPY . /go/src/github.com/docker/docker
+
```

Comment:
Why do these files need to be available here? This would make debug builds much slower so probably need to find another way.